src/Idephix/Console/Application.php

```diff
@@ -4,16 +4,22 @@
 
 use Symfony\Component\Console\Application as BaseApplication;
 use Symfony\Component\Console\Input\InputOption;
+use Idephix\Console\Command\HelpCommand;
```

Comment:
Is this class needed? Our `\Idephix\Console\Command\HelpCommand` seems identical to `\Symfony\Component\Console\Command\HelpCommand`, isn't it?